### DISCUSSION TITLE
Does OBS support the SRT protocol on Ubuntu 18.04 and above?

### DISCUSSION THREAD

**Discussion by Gigalomanicx** [obsproject/obs-studio#8448] [General]
I recently used Obs 27 and 29 on Ubuntu 18.04 and 20.04, respectively. When I tried to stream SRT, I encountered the problem of Protocol Not Found. And I compiled the source code of libsrt in Ubuntu. Can the SRT protocol be used on Ubuntu? I can't find the relevant documents. I only know that OBS supports the SRT protocol after 25.

[18:17:05]Couldn't open 'srt://xxx.xxx.xxx.xxx:10080?streamid=#!::r=live/livestream,latency=20,m=request', Protocol not found.

I tried to use Obs 29 on Windows to push the same srt connection, and it was successful.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
